frontend/test/metabase/visualizations/visualizations/PieChart.unit.spec.js

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from "react";
+import { thaw } from "icepick";
+import userEvent from "@testing-library/user-event";
+import { renderWithProviders, screen, waitFor } from "__support__/ui";
+
+import {
+  SAMPLE_DATABASE,
+  ORDERS,
+  metadata,
+} from "__support__/sample_database_fixture";
+import ChartSettings from "metabase/visualizations/components/ChartSettings";
+
+import Question from "metabase-lib/Question";
+
+const setup = () => {
+  const Container = () => {
+    const [question, setQuestion] = useState(
+      new Question(
+        {
+          display: "pie",
+          dataset_query: {
+            type: "query",
+            query: {
+              "source-table": ORDERS.id,
+              aggregation: [["count"]],
+              breakout: [
+                ["field", ORDERS.CREATED_AT.id, { "temporal-unit": "year" }],
+              ],
+            },
+          },
+          database: SAMPLE_DATABASE.id,
+          visualization_settings: {},
+        },
+        metadata,
+      ),
+    );
+
+    const onChange = update => {
+      setQuestion(q => {
+        const newQuestion = q.updateSettings(update);
+        return new Question(thaw(newQuestion.card()), metadata);
+      });
+    };
+
+    return (
+      <ChartSettings
+        onChange={onChange}
+        series={[
+          {
+            card: question.card(),
+            data: {
+              rows: [
+                ["2016-01-01T00:00:00-04:00", 500],
+                ["2017-01-01T00:00:00-04:00", 1500],
+              ],
+              cols: question.query().columns(),
+            },
+          },
+        ]}
+        initial={{ section: "Data" }}
+        question={question}
+      />
+    );
+  };
+
+  renderWithProviders(<Container />);
+
+  //Append mocked style for .hide class
+  const mockedStyle = document.createElement("style");
```

Comment:
Since you don't clean this up, I assume the last test will have 5 style elements. How about moving to `beforeAll`?

Reply:
Haha, you are probably right. Good suggestion!